test/cmdUtil.js

```diff
@@ -0,0 +1 @@
+require('../lib/cmdUtil.js').generateNewToken();
```

Comment:
Ah, I see, this is meant to be used to start the generation process, not as a test case? This feels a bit off to me... See my [comment](https://github.com/UnbounDev/node-strava-v3/pull/35#issuecomment-294180282), maybe it makes sense to move utility scripts into their own folder.